README.md

```diff
@@ -28,7 +28,14 @@ For every major Minecraft version, two branches exist:
 
 ### Building and setting up a development environment
 
-This mod uses [Project Lombok](http://projectlombok.org/) -- an annotation processor that allows us you to generate constructors, getters and setters using annotations -- to speed up recurring tasks and keep part of our codebase clean at the same time. Because of this it is advised that you install a plugin for your IDE that supports Project Lombok. Should you encounter any weird errors concerning missing getter or setter methods, it's probably because your code has not been processed by Project Lombok's processor. A list of Project Lombok plugins can be found [here](http://projectlombok.org/download.html).
+To get started, you will need a [Maven token](https://github.com/settings/tokens/new?scopes=read:packages&description=GPR%20for%20Gradle) so that gradle can fetch dependencies.
+Once you have this, use it by setting the environment variables `MAVEN_USERNAME` (to your github username), and `MAVEN_KEY` (to your newly acquired token).
```

Comment:
Setting these variables via `gradle.properties` is more user-friendly, so I would suggest documenting this approach (first). Environment variables are usually just used for the CI.

Reply:
I'll need to get you to explain to me exactly how you do that - because when I tried it that way, it didn't work.

Reply:
This should work:

<img width="404" alt="afbeelding" src="https://user-images.githubusercontent.com/440384/195001182-dc8edfac-c760-41fc-a984-e476e46020c1.png">

Reply:
That's exactly what I tried... Oh well. I'll add that too, followed by the variables fallback.